editor/ScreenLayers/ProjectMenu.cs

```diff
@@ -550,7 +552,7 @@ public override void Update(bool isTop, bool isCovered)
             if (Manager.GetContext<Editor>().IsFixedRateUpdate)
             {
                 timeButton.Text = Manager.GetContext<Editor>().InputManager.Alt ?
-                    $"{storyboardPosition.X:000}, {storyboardPosition.Y:000}" :
+                    $"{storyboardPosition.X:###}, {storyboardPosition.Y:###}" :
```

Comment:
What's that for?

Reply:
This is mainly for consistency in display with the copy output as `000` format will display the coordinates as for example `273, 097`, where the leading 0 does not really make any sense while `###` changes the display to `273, 97`.
I don't mind scrapping that part though, don't feel strongly about it.